### ISSUE TITLE
Steam layouts and dolphin config don't seem to work

### ISSUE THREAD

**Issue by strauzen** [EmuDeck/emudeck.github.io#39]
After updating emudeck and all the emulators on my SteamDeck I have installed I noticed that my wii-u games don't seem to work. No imput detected and no hotkeys to save and load states.

When I dig a bit deeper it seems that the inputs in the dolphin profiles are mapped to the keyboard instead of the SteamDeck buttons. Moreover I don't seem to have a controller layout in steam that has the hotkeys assigned for saving, loading state and other actions.

Any idea what might be going wrong and what I can do? Before the update today everything was working great.

**Comment by yeager** [comment]
Yeah, you have to reset the Dolphin configuration after every update
